app/views/simplified-account/settings/email-notifications/collect-email-page.njk

```diff
@@ -28,23 +28,23 @@
           items: [
             {
                 value: emailCollectionModes.mandatory,
-                text: 'On – as a mandatory field',
+                html: '<b>On – as a mandatory field</b>',
```

Comment:
you can add `govuk-!-font-weight-bold` to `classes` instead of using html `<b>` tags. See nunjucks macro options for [summary list component](https://design-system.service.gov.uk/components/summary-list/)